### ISSUE TITLE
Update error on execution 2.17.2

### ISSUE THREAD

**Issue by franckadil** [RocketChat/Rocket.Chat.Electron#1460]
## My Setup
- Operating System: Windows 10
- App Version: 2.17.2
- Installation type: exe

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description

The console outputs an error on yarn start: `Redirect was cancelled`
### Current Behavior

```
src/preload.js → app/preload.js...
created app/preload.js in 680ms
[17:43:59] Finished 'build:bundle' after 3.52 s
[17:43:59] Finished 'build' after 3.52 s
[17:43:59] Starting 'watch'...
[17:43:59] Starting 'start:electron'...
[17:43:59] [gulp-execa] electron .

 Checking for update
Error: Error: Redirect was cancelled
    at SimpleURLLoaderWrapper.<anonymous> (electron/js2c/browser_init.js:2561:23)
    at SimpleURLLoaderWrapper.emit (events.js:203:13)
    at SimpleURLLoaderWrapper.EventEmitter.emit (domain.js:476:20)
    at SimpleURLLoaderWrapper.topLevelDomainCallback (domain.js:131:23)
Error: Redirect was cancelled
```

### Expected Behavior

The console output should be free of error, and updates should be checked correctly, based on electron-auto updater.

**Comment by franckadil** [comment]
@tassoevan  It's a regression in Electron 7.1.3 in how aborted redirect requests are handled that broke updates for all apps using electron-updater. I also tested this with:
electron@7.1.7
electron-updater@4.2.0
electron-builder@21.2.0

The good news is that it looks like a fix might have been merged into master. Seems important enough to authorize a regression test for it. It's not sure what is their process to get it backported to 7 and released. electron/electron@ed6a387. But I hope this ticket can be useful to keep track of the release.
